### DISCUSSION TITLE
Unicorn achievement

### DISCUSSION THREAD

**Discussion by cesarreyes3** [Flet/rejected-github-profile-achievements#4] [Show and tell]
@Flet How about unicorn as rejected achievement
![image](https://user-images.githubusercontent.com/109559100/181369670-7ee6d6a2-231f-4cbd-a4ab-844e3a4fb348.png)
Make your profile imposible to watch
Example: https://github.com/paledega

**Comment by Flet** [comment]
Hah, never see this happen before on a profile!

Making the badge itself say "badge taking too long to load" might be funny. Maybe with a half loaded unicorn? :D

**Reply by cesarreyes3** [reply]
Well, he did make a script to bring down github
![image](https://user-images.githubusercontent.com/109559100/181386524-c9d45575-7d29-403f-a398-bca8e2e50ec9.png)
Was hard to find

**Comment by jaythomas** [comment]
I think the real joke here is how much GitHub goes offline. So the unicorn error is very familiar to many of us. Not sure of a way you can spin that into a funny badge though.
